app/views/news_tematica/news_tematicas/show.html.haml

```diff
@@ -1,9 +1,10 @@
-:sass
-  .cuentaclics
-    background: #FF3
-    color: black
-    padding: 4px
-    margin-right: 8px
+:css
+  .cuentaclics {
+    background: #FF3;
```

Comment:
Colors in css should be 'lowercase' `#ff3`, if you want you can leave it, it's not a big problem.